### PR TITLE
i18n(fr): Update CloudFlare adapter docs for v10 from 7210e31

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Update CloudFlare adapter docs for v10 from 7210e31

#### Related issues & labels (optional)

- Closes #
- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/integrations-guide/cloudflare.mdx [hunk 1]
```diff
@@ -79,164 +79,133 @@ export default defineConfig({
...
+			experimentalJsonConfig: true,
+			persist: './.cache/wrangler/v3',
+		},
+	}),
 });
 ```
+### routes.extend
 
-### imageService
+Cette option vous permet d'ajouter ou d'exclure des motifs personnalisés (par exemple `/fonts/*`) au fichier [`_routes.json`](#_routesjson-personnalisé) généré qui détermine quels itinéraires sont générés à la demande. Cela peut être utile si vous avez besoin d'ajouter des modèles d'itinéraires qui ne peuvent pas être générés automatiquement, ou d'exclure des itinéraires pré-rendus.
```
Comment (pioupia) [inline src/content/docs/fr/guides/integrations-guide/cloudflare.mdx:171]:
```suggestion
Cette option vous permet d'ajouter ou d'exclure des motifs personnalisés (par exemple `/fonts/*`) au fichier [`_routes.json`](#_routesjson-personnalisé) généré qui détermine quelles routes sont générées à la demande. Cela peut être utile si vous avez besoin d'ajouter des modèles de route qui ne peuvent pas être générés automatiquement, ou d'exclure des routes pré-rendues.
```

File: src/content/docs/fr/guides/integrations-guide/cloudflare.mdx [hunk 2]
```diff
@@ -79,164 +79,133 @@ export default defineConfig({
...
 
-<p>
+Vous trouverez plus d'informations sur les modèles d'itinéraires personnalisés dans la [documentation de routing de Cloudflare](https://developers.cloudflare.com/pages/functions/routing/#functions-invocation-routes). Les routes spécifiées ne sont pas automatiquement dédupliquées et seront ajoutées telles quelles aux routes existantes.
```
Comment (pioupia) [inline src/content/docs/fr/guides/integrations-guide/cloudflare.mdx:173]:
```suggestion
Vous trouverez plus d'informations sur les modèles de routes personnalisées dans la [documentation de routing de Cloudflare](https://developers.cloudflare.com/pages/functions/routing/#functions-invocation-routes). Les routes spécifiées ne sont pas automatiquement dédupliquées et seront ajoutées telles quelles aux routes existantes.
```

**Review by pioupia — CHANGES_REQUESTED** [review]

**Comment by thomasbnt** [comment]
Merci @pioupia pour la review!

**Review by pioupia — APPROVED** [review]
LGTM!

**Comment by yanthomasdev** [comment]
Heads up we got a few updates here, could you apply these changes to the PR? 🙌 https://github.com/withastro/docs/commits/main/src/content/docs/en/guides/integrations-guide/cloudflare.mdx?since=2024-03-24T17:21:37.000Z

**Comment by thomasbnt** [comment]
> Heads up we got a few updates here, could you apply these changes to the PR? 🙌 [`main`/src/content/docs/en/guides/integrations-guide/cloudflare.mdx?since=2024-03-24T17:21:37.000Z (commits)](https://github.com/withastro/docs/commits/main/src/content/docs/en/guides/integrations-guide/cloudflare.mdx?since=2024-03-24T17:21:37.000Z)

Just seen your mesage @yanthomasdev, don't hesitate to mention me. I pushed modifications!

**Review by yanthomasdev — APPROVED** [review]
LGTM! Thanks 🙌

**Pull request merged**